tests/Traits/AttributeHelper.php

```diff
@@ -47,13 +47,6 @@ public function getAttributes(ProductEntity $productEntity, string $integrationT
             ],
         );
 
-        foreach ($productEntity->options ?? [] as $option) {
```

Comment:
You could move it to a `getOptions()` function, so you can make the assertion like we do for the attributes. (Check the usage of `getAttributes()`)